src/main/kotlin/racingcar/model/Car.kt

```diff
@@ -0,0 +1,22 @@
+package racingcar.model
+
+data class Car(val name: String, private val distance: Int) {
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
data class Car(val name: String, distance: Int) {
```

Reply:
수정하였습니다.
[2c449c725db00d45e9685388ddb76ce8843b23ea]